generated/nidaqmx/_task_modules/channels/ao_channel.py

```diff
@@ -1820,8 +1821,9 @@ def ao_output_impedance(self):
     @property
     def ao_output_type(self):
         """
-        :class:`nidaqmx.constants.UsageTypeAO`: Indicates whether the
-            channel generates voltage,  current, or a waveform.
+        :class:`nidaqmx.constants.AOOutputChannelType`: Indicates
```

Comment:
Its still `UsageTypeAO` in the constants.py PR #215. I don't think this is correct.

Reply:
Updated.